### ISSUE TITLE
Improve interface for ending polls

### ISSUE THREAD

**Issue by kittykat** [vector-im/element-web#20300]
### Your use case

As a poll creator, I would like to have the most common/useful interaction options available to me with the fewest clicks.

I feel like "end poll" should be one of the options you can see on hovering over the poll. There is prior art for this with image download option being added as the first one.

![Screenshot from 2021-12-21 11-26-46](https://user-images.githubusercontent.com/51663/146923153-a5317e5d-53c0-42bd-a302-9bd98924456b.png)

**Comment by HarHarLinks** [comment]
> I feel like "end poll" should be one of the options you can see on hovering over the poll.

I agree and would go further: The end poll action would still be well hidden even in the hovering popup. In my opinion it would be even better if the poll creator had a very obvious button like this

![image](https://user-images.githubusercontent.com/2803622/154096432-b4772efc-8308-45b8-8a63-77691b2143a0.png)
